_content/monographs/the-discourse-on-right-view_nyanamoli.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "The Discourse on Right View: The Sammādiṭṭhi Sutta and its Commentary"
+authors:
+  - nyanamoli
+editor: bodhi
+external_url: "https://www.bps.lk/olib/wh/wh377_Nyanamoli_Discourse-on-Right-View--Sammaditthi-Sutta.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/14KeRv_PMUq8Naaov6IgWjsx9-hNvYxGg/view?usp=drive_link"
+formats: [pdf]
+status: featured
+course: view
+tags:
+  - pali-texts
+  - dhamma
+oclc: 33109236
+year: 1991
+publisher: Buddhist Publication Society
+address: "Kandy"
+volume: "Wheel Publication"
+pages: 105
```

Comment:
Because these PDFs have extra small page sizes, I've been dividing their page counts by 2 (assuming people, if they were to print these out, would print two pdf pages per letter-sized page).